packages/command/src/sfdxFlags.ts

```diff
@@ -228,7 +228,10 @@ function buildStringArray(kind: flags.Kind, options: flags.Array<string>): flags
   return option(kind, rest, (val) => {
     // eslint-disable-next-line no-useless-escape
     const regex = /\"(.*?)\"|\'(.*?)\'|,/g;
```

Comment:
@peternhale it doesn't look like stringArray supported custom delimiters properly (adding a map option pushes it into the buildMappedArray method which does.

Reply:
The issue for me is there are no tests that create a array flag that defines a custom delimiter.

Reply:
added the tests, and fixed the "works with maps AND string problems" over here
https://github.com/forcedotcom/cli-packages/pull/139